src/Func.jl

```diff
@@ -13,7 +13,7 @@ const OR = Func(|, 2)
 const XOR = Func($, 2)
 const NOT = Func(~, 1)
 const ZERO = Func(() -> convert(BitString, 0), 0, "0")
-const ONE = Func(() -> convert(BitString, ~0), 0, "1")
+const ONE = Func(() -> ~convert(BitString, 0), 0, "1")
```

Comment:
This was done because in Julia 0.4 the `~0` syntax resulted in a loss of precision error.